client/my-sites/site-settings/main.jsx

```diff
@@ -104,6 +105,7 @@ export class SiteSettingsComponent extends Component {
 			<Main className="site-settings">
 					<SidebarNavigation />
 					<SiteSettingsNavigation site={ site } section={ section } />
+					<QueryProductsList />
```

Comment:
I think you'll be querying the product list more times than you need here. I would move this to the `GuidedTransfer` component so that we only query the product list when we need to display the cost.